pxf/build.gradle

```diff
@@ -286,6 +286,29 @@ project('pxf-hive') {
     }
 }
 
+project('pxf-json') {
+	dependencies {
+		compile(project(':pxf-hdfs'))
+		compile "org.apache.commons:commons-lang3:3.0"
+
+		testCompile 'pl.pragmatists:JUnitParams:1.0.2'
+		configurations {
+			// Remove hive-exec from unit tests as it causes VerifyError
+			testRuntime.exclude module: 'hive-exec'
```

Comment:
I mean the exclude part - it refers to hive module that is not a dependency for this plugin.
Could you try to remove the configurations block and run make to see if it works?